cmd/release/config/config.go

```diff
@@ -74,3 +91,89 @@ func read(r io.Reader) (*Config, error) {
 	}
 	return &c, nil
 }
+
+func (c *Config) String() (string, error) {
+	b, err := json.MarshalIndent(c, "", " ")
+	if err != nil {
+		return "", err
+	}
+	return string(b), nil
+}
+
+func OpenOnEditor() error {
+	confPath, err := DefaultConfigPath()
+	if err != nil {
+		return err
+	}
+	cmd := exec.Command(textEditorName(), confPath)
+	cmd.Stdin = os.Stdin
+	cmd.Stdout = os.Stdout
+	return cmd.Run()
+}
+
+func GenConfig() error {
```

Comment:
```suggestion
func Generate() error {
```

Let's rename this to avoid the stutter of config.GenConfig.